R/utils.R

```diff
@@ -466,7 +469,7 @@ fmt_txt <- function(
 
 #' @rdname fmt_txt
 #' @method as.character fmt_txt
-#' @param x an openxlsx2 fmt_txt string
+# #' @param x an openxlsx2 fmt_txt string
```

Comment:
to avoid R CMD Check warnings

Reply:
We should probably comment this, otherwise I or whoever comes across this function in the future, will believe it's a typo and remove the first #